Services/StopPointManager.php

```diff
@@ -43,6 +43,17 @@ private function initTitle($externalCoverageId)
         );
     }
 
+    private function initCity($externalCoverageId)
+    {
+        $this->stopPoint->setCity(
+            $this->navitia->getStopPointCity(
+                $externalCoverageId,
+                $this->stopPoint->getExternalId(),
+                ["depth" => 1, "show_codes" => "true"]
```

Comment:
Timetable requires php >=5.3.2 so [  ] notation is not allowed.
user array() notation